graphics/src/ColladaLoader.cc

```diff
@@ -2337,9 +2363,15 @@ void ColladaLoaderPrivate::LoadTriangles(tinyxml2::XMLElement *_trianglesXml,
     }
     else if (semantic == "TEXCOORD")
     {
-      // we currently only support one set of UVs
-      this->LoadTexCoords(source, texcoords, texDupMap);
-      inputs[TEXCOORD].insert(ignition::math::parseInt(offset));
+      int offsetInt = ignition::math::parseInt(offset);
+      if (inputs[TEXCOORD].find(offsetInt) == inputs[TEXCOORD].end())
+      {
+        unsigned int set = ignition::math::parseInt(
+            trianglesInputXml->Attribute("set"));
```

Comment:
Same comment about whether we need to support empty `set` attributes

Reply:
added check. 50c0e2e